rollup.config.mjs

```diff
@@ -1,7 +1,6 @@
 // this config rolls up all the types in the project to a single declaration (d.ts) file.
 // we do NOT use rollup to build (we use esbuild for that)
 
-import alias from '@rollup/plugin-alias';
```

Comment:
Is this no longer being used? Can we remove it as a dependency?

Reply:
yep